Introduction/Getting started/Installation/task.md

```diff
@@ -9,6 +9,27 @@ Let’s start your Rust journey! There’s a lot to learn, but every journey sta
 
 In this lesson and throughout the course, we’ll show some commands used in the terminal. Lines that you should enter in a terminal all start with `$`. You don’t need to type in the `$` character; it indicates the start of each command. Lines that don’t start with `$` typically show the output of the previous command. Additionally, PowerShell-specific examples will use <code>&gt;</code> rather than `$`.
      
+### Rust Installation
+#### Unix
+1. In Terminal run the command to download Rust toolchain installer:
+```text
+$ curl https://sh.rustup.rs -sSf | sh
+```
+2. Now run this:
+```text
+$ source $HOME/.cargo/env
+```
+3. Restart your IDE.
+4. Setup Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").
+5. You're good to go!
+
+#### Windows
+1. Download rustup installer from https://win.rustup.rs/x86_64.
+2. Run it and follow the onscreen instructions.
+3. Restart your IDE.
+4. Setup Rust toolchain (press "Setup toolchain" in the pop-up "No Rust toolchain configured").
+5. You're good to go!
+
 ### Verification of Rust installation
 
 You need to set up the environment in a system below are few things which you should check before the work.
```

Comment:
I believe the default case would be to install the rust, so I suggest changing the "check before the work" to "do before the work" and move this sentence to the very top (before the "Unix" header)